.run/CaskCodeGen.run.xml

```diff
@@ -0,0 +1,25 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="CaskCodeGen" type="Application" factoryName="Application">
+    <envs>
+      <env name="PACKAGE" value="test.demo" />
+      <env name="DEBUG" value="false" />
+    </envs>
+    <option name="MAIN_CLASS_NAME" value="org.openapitools.codegen.OpenAPIGenerator" />
+    <module name="openapi-generator-cli" />
+    <option name="PROGRAM_PARAMETERS" value="generate -g scala-cask -o $PROJECT_DIR$/samples/scala-cask -i $PROJECT_DIR$/modules/openapi-generator/src/test/resources/3_1/petstore.yaml -c ./bin/config.yml" />
+    <extension name="coverage">
+      <pattern>
+        <option name="PATTERN" value="io.swagger.codegen.*" />
+        <option name="ENABLED" value="true" />
+      </pattern>
+    </extension>
+    <extension name="software.aws.toolkits.jetbrains.core.execution.JavaAwsConnectionExtension">
+      <option name="credential" />
+      <option name="region" />
+      <option name="useCurrentConnection" value="false" />
+    </extension>
+    <method v="2">
+      <option name="Make" enabled="true" />
+    </method>
+  </configuration>
+</component>
```

Comment:
what's the purpose of this file?

Reply:
It was a run profile for testing -- something I can just take out